client/asset/dcr/simnet_test.go

```diff
@@ -58,18 +63,22 @@ func mineAlpha() error {
 	return exec.Command("tmux", "send-keys", "-t", "dcr-harness:0", "./mine-alpha 1", "C-m").Run()
 }
 
-func tBackend(t *testing.T, name string, blkFunc func(string, error)) (*ExchangeWallet, *dex.ConnectionMaster) {
+func tBackend(t *testing.T, name string, isInternal bool, blkFunc func(string, error)) (*ExchangeWallet, *dex.ConnectionMaster) {
 	t.Helper()
 	user, err := user.Current()
 	if err != nil {
 		t.Fatalf("error getting current user: %v", err)
 	}
-	cfgPath := filepath.Join(user.HomeDir, "dextest", "dcr", name, name+".conf")
-	settings, err := config.Parse(cfgPath)
-	if err != nil {
-		t.Fatalf("error reading config options: %v", err)
+	settings := make(map[string]string)
+	if !isInternal {
+		cfgPath := filepath.Join(user.HomeDir, "dextest", "dcr", name, name+".conf")
+		var err error
+		settings, err = config.Parse(cfgPath)
+		if err != nil {
+			t.Fatalf("error reading config options: %v", err)
+		}
+		settings["account"] = "default"
```

Comment:
Can actually drop this now. We're doing [this](https://github.com/decred/dcrdex/blob/4ab15ac46abf273caa55f898360c07b0a4a8a0c4/client/asset/dcr/dcr.go#L784-L787).

Reply:
Removed `settings["account"] = "default"`